Examples/linux/vdm_dump.cpp

```diff
@@ -28,6 +28,8 @@
 
 */
 
+#ifndef ESP32
```

Comment:
This example is for Linux.
Should we add #ifndef for all none-linux platforms then. I think no